src/ResourceManagement/Resource/ResourceManagement/Generated/Models/DeploymentPreFlightResource.cs

```diff
@@ -61,19 +61,5 @@ public DeploymentPreFlightResource()
         {
             this.DependsOn = new LazyList<string>();
         }
-        
-        /// <summary>
-        /// Initializes a new instance of the DeploymentPreFlightResource class
-        /// with required arguments.
-        /// </summary>
-        public DeploymentPreFlightResource(string location)
```

Comment:
@markcowl this? https://github.com/Azure/azure-sdk-for-net/commit/c860ffda5d6190163cdd7310088c2cfb050027e3